pkg/pillar/go.sum

```diff
@@ -1456,6 +1456,8 @@ github.com/lf-edge/eve-api/go v0.0.0-20250227172635-d65ee400e16d h1:3GinRDoSMhMR
 github.com/lf-edge/eve-api/go v0.0.0-20250227172635-d65ee400e16d/go.mod h1:ot6MhAhBXapUDl/hXklaX4kY88T3uC4PTg0D2wD8DzA=
 github.com/lf-edge/eve-api/go v0.0.0-20250228142514-8c31fa62d594 h1:K9Vu2Mb69EBNnZYsZR3Y+X6fNvYHgj2ePl2vNzhepAs=
 github.com/lf-edge/eve-api/go v0.0.0-20250228142514-8c31fa62d594/go.mod h1:ot6MhAhBXapUDl/hXklaX4kY88T3uC4PTg0D2wD8DzA=
+github.com/lf-edge/eve-api/go v0.0.0-20250304184518-532c94981505 h1:J4pEkDOnO3o+MVSsL5Xr6lcO/fYApcALbVpfUONijp0=
```

Comment:
Try running `go mod tidy` - it should remove the previous eve-api versions from go.sum